src/main/java/com/o19s/ubi/action/UserBehaviorInsightsActionFilter.java

```diff
@@ -46,7 +46,7 @@ public class UserBehaviorInsightsActionFilter implements ActionFilter {
 
     private final Client client;
     private final ThreadPool threadPool;
-    private final EventManager eventManager;
```

Comment:
I renamed this from `EventManager` to `DataManager` because it manages more than events - it also manages query requests.